test/extended/router/gatewayapicontroller.go

```diff
@@ -0,0 +1,171 @@
+package router
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	g "github.com/onsi/ginkgo/v2"
+	o "github.com/onsi/gomega"
+
+	exutil "github.com/openshift/origin/test/extended/util"
+	e2e "k8s.io/kubernetes/test/e2e/framework"
+	admissionapi "k8s.io/pod-security-admission/api"
+
+	sailv1 "github.com/istio-ecosystem/sail-operator/api/v1"
```

Comment:
I don't think we should import this package just for `sailv1.IstioReasonHealthy`, we need to update go.mod and vendor to make it work.